src/eterna/pose2D/PoseField.ts

```diff
@@ -185,6 +186,8 @@ export default class PoseField extends ContainerObject implements KeyboardListen
         if (Flashbang.app.isControlKeyDown) {
             return;
         }
+        // kkk ignore mouse event on 3d view
+        if (Mol3DGate.scope && Mol3DGate.scope.isOver3DCanvas) return;
```

Comment:
Yeah stuff like this should be able to be avoided by just setting the 3D view's container to be interactive so that stuff under it doesn't get mouse events